app/models/graph-controller.ts

```diff
@@ -81,10 +81,20 @@ export class GraphController {
         }
 
         // TODO: ...like
+        let likeVal: ElementValue | undefined = undefined;
+        if (like) {
+            const bridge = this.bridges.getB(like);
+            if (bridge) {
+                likeVal = bridge.core;
+            }
+        }
 
-        return validateEdge(this.plugin, source, destination, undefined);
-    }
+        if (!likeVal) {
+            likeVal = new ElementValue(this.activeEdgeType, this.core.environment);
```

Comment:
This is concerning because once we're done with ElementValue's we need to explically remove them from the environment. The like parameter should probably be just ElementType | ElementValue | undefined ?